default.nix

```diff
@@ -67,16 +67,7 @@ let
   latex = pkgs.callPackage ./nix/latex.nix {};
   sources = import ./nix/sources.nix;
 
-  # easy-purescript-nix has some kind of wacky internal IFD
-  # usage that breaks the logic that makes source fetchers
-  # use native dependencies. This isn't easy to fix, since
-  # the only places that need to use native dependencies
-  # are deep inside, and we don't want to build the whole
-  # thing native. Fortunately, we only want to build the
-  # client on Linux, so that's okay. However, it does
-  # mean that e.g. we can't build the client dep updating
-  # script on Darwin.
```

Comment:
Keep the comment, please!

Reply:
This should now work on Darwin, hence I killed the comment.

Reply:
What makes you think that?

Reply:
I talked with justin about it, and also read the source of easy-purescript-nix: https://github.com/justinwoo/easy-purescript-nix/blob/6be3f48f339034a58b1b1ae997ace534cf459826/default.nix#L1 - where that value is propagated.

Reply:
I'll believe it when I see it :) This definitely did not work previously, so I'm suspicious that it now "works" without anything changing.

Reply:
I did change something, see line below.

Reply:
That doesn't change anything: `callPackage` already passes in the `pkgs` argument as itself.

Reply:
And that line is not the place where the dodgy stuff happens, it's much deeper in.